src/storagemanager/src/main/java/org/vpac/ndg/task/TileBuilder.java

```diff
@@ -94,23 +97,37 @@ public void initialise() throws TaskInitialisationException {
         return command;
     }
 
-    
-    
     @Override
-    public void execute(Collection<String> actionLog) throws TaskException {
-        
+    public void execute(Collection<String> actionLog, IProgressCallback progressCallback) throws TaskException {
+
         List<String> command = prepareCommand();
 		actionLog.add(StringUtils.join(command, " "));
         try {
-            commandUtil.start(command);
-        } catch (ProcessException e) {
-            throw new TaskException(getDescription(), e);
-        } catch (InterruptedException e) {
-            throw new TaskException(getDescription(), e);
+            ProcessBuilder pb = new ProcessBuilder(command);
```

Comment:
Does switching to `ProcessBuilder` also add error handling in the case that the tiles fail to generate?